src/video_core/engines/maxwell_3d.h

```diff
@@ -1152,6 +1157,9 @@ class Maxwell3D final {
     /// Parameters that have been submitted to the macro call so far.
     std::vector<u32> macro_params;
 
+    std::array<bool, Regs::NUM_REGS> must_process_dirty;
```

Comment:
```suggestion
    std::array<bool, Regs::NUM_REGS> must_process_dirty{};
```

otherwise this initializes to an inconsistent state.

These two arrays are also missing a documentation comment.